content/en/_index.md

```diff
@@ -17,65 +17,73 @@ description: "Documentation related to EGI activities"
     Learn More about EGI <i class="fas fa-arrow-alt-circle-right ml-2"></i>
   </a>
 
-  <a class="btn btn-lg btn-secondary mr-3 mb-4"
-     href="{{< relref "/about/contributing" >}}">
-    Contribute <i class="fab fa-github ml-2 "></i>
-  </a>
-  <p class="lead mt-5">EGI Documentation for users and service providers.</p>
+<a class="btn btn-lg btn-success mr-3 mb-4"
+href="{{< relref "/about/contributing" >}}"> Contribute
+<i class="fab fa-github ml-2 "></i> </a>
+
+  <p class="lead mt-2">EGI Documentation for users and service providers.</p>
 
-  {{< blocks/link-down color="info" >}}
+{{< blocks/link-down color="info" >}}
 
 </div>
 {{< /blocks/cover >}}
 
 <div class="mx-auto">
 {{< blocks/section color="primary" type="features">}}
 
-{{% blocks/feature icon="fa-lightbulb" title="Users" url="/users" %}}
-Contains step-to-step documentation on how to use the EGI services.
-{{% /blocks/feature %}}
+{{% blocks/feature icon="fa-user-friends" title="User Guide" url="/users" %}}
+Contains documentation of the EGI services and step-by-step guides on how to
+use them. {{% /blocks/feature %}}
 
-{{% blocks/feature icon="fa-lightbulb" title="Providers" url="/providers" %}}
-Depicts how to join the EGI infrastructure as a service providers offering
-innovative services to the European Research Area. {{% /blocks/feature %}}
+{{% blocks/feature icon="fa-users-cog" title="Service Providers"
+url="/providers" %}} Explains how to join the EGI infrastructure as a service
+provider to offer your innovative services. {{% /blocks/feature %}}
 
-{{% blocks/feature icon="fa-lightbulb" title="Internal services"
-    url="/internal" %}}
-Hosts the documentation of the EGI services enabling the federation.
-{{% /blocks/feature %}}
+{{% blocks/feature icon="fa-cogs" title="Internal Services" url="/internal" %}}
+Hosts the documentation of the EGI services that power the
+cloud services of EGI Federation. {{% /blocks/feature %}}
 
 {{< /blocks/section >}}
 
 </div>
 
 {{< blocks/section color="dark" type="features">}}
+
 <!-- markdown-link-check-disable -->
-{{% blocks/feature icon="fa-lightbulb" title="Learn more about EGI!"
+
+{{% blocks/feature icon="fa-lightbulb" title="Learn about EGI"
     url="https://www.egi.eu" %}}
+
 <!-- markdown-link-check-enable-->
-The EGI Federation is supporting lots of different user communities.
-{{% /blocks/feature %}}
+
+The EGI Federation is supporting many different user communities and open
+science projects. {{% /blocks/feature %}}
 
 <!-- markdown-link-check-disable -->
-{{% blocks/feature icon="fab fa-github" title="Contributions welcome!"
+
+{{% blocks/feature icon="fab fa-github" title="Contribute!"
     url="https://docs.egi.eu/about/contributing" %}}
+
 <!-- markdown-link-check-enable-->
-We do a [Pull Request](https://github.com/EGI-Federation/documentation/pulls)
-contributions workflow on **GitHub**. New users are always welcome!
+
+We do a contributions workflow on GitHub. New users are always welcome!
```

Comment:
This reads weird. Like missing a verb maybe?

Reply:
It's what is on the site now. Updated.